City-of-Helsinki/haitaton-ui
# HAI-2569 Fetch banners from API

# Description

Instead of using environment variables for INFO/WARNING/ERROR banners, fetch them from backend.

### Jira Issue: https://helsinkisolutionoffice.atlassian.net/browse/HAI-2569

## Type of change


# Instructions for testing

* Use the latest backend dev version
* Add some banner texts into the database by hand (see https://helsinkisolutionoffice.atlassian.net/wiki/spaces/HAI/pages/8297807895/Yll+pito-ohje+maintenance+guide)
* The banners should work
* Try to modify and/or delete the banners in database and check that the changes are reflected in the UI

# Checklist:

- [x] I have ran the tests myself (if applicable)
- [ ] I have made necessary changes to the documentation, link to confluence
      or other location:

# Other relevant info

Please describe here if there is e.g. some requirements for this change or
other info that the tester/user needs to know.
